src/components/SiteFooter/index.jsx

```diff
@@ -0,0 +1,214 @@
+              <Icon className={['fa', 'fa-google-plus-square', 'fa-2x']} screenReaderText={`Follow ${siteName} on Google+`} />
+            </a>
+          </li>
+          <li>
+            <a href={redditUrl} title="Reddit" rel="noopener noreferrer" target="_blank">
+              <Icon className={['fa', 'fa-reddit-square', 'fa-2x']} screenReaderText={`Subscribe to the ${siteName} subreddit`} />
+            </a>
+          </li>
+        </ul>
+      );
+    }
+    return socialLinks;
+  }
+
+  renderMobileLinks() {
+    const {
+      siteName,
+      showMobileLinks,
+      appleAppStoreUrl,
+      googlePlayUrl,
+    } = this.props;
+    let mobileLinks = null;
+    if (showMobileLinks) {
+      mobileLinks = (
+        <ul className="d-flex flex-row justify-content-between list-unstyled max-width-264 p-0 mb-5">
+          <li>
+            <a href={appleAppStoreUrl} rel="noopener noreferrer" target="_blank">
+              <img
+                className="max-height-39"
+                alt={`Download the ${siteName} mobile app from the Apple App Store`}
+                src="https://prod-edxapp.edx-cdn.org/static/images/app/app_store_badge_135x40.d0558d910630.svg"
+              />
+            </a>
+          </li>
+          <li>
+            <a href={googlePlayUrl} rel="noopener noreferrer" target="_blank">
+              <img
+                className="max-height-39"
+                alt={`Download the ${siteName} mobile app from Google Play`}
+                src="https://prod-edxapp.edx-cdn.org/static/images/app/google_play_badge_45.6ea466e328da.png"
+              />
+            </a>
+          </li>
+        </ul>
+      );
+    }
+    return mobileLinks;
+  }
+
+  render() {
+    const {
+      siteName,
+      openSourceUrl,
+      termsOfServiceUrl,
+      privacyPolicyUrl,
+      contactUrl,
+      supportUrl,
+    } = this.props;
+    return (
+      <footer
+        role="contentinfo"
+        aria-label="Page Footer"
+        className="footer d-flex justify-content-center border-top py-3 px-4"
+      >
+        <div className="max-width-1180 d-grid">
+          <div className="area-1">
+            <Hyperlink destination={this.renderMarketingSiteUrl('/')} content={this.renderSiteLogo()} aria-label={`${siteName} Home`} />
+          </div>
+          <div className="area-2">
+            <h2>{siteName}</h2>
+            <ul className="list-unstyled p-0 m-0">
+              <li><a href={this.renderMarketingSiteUrl('/about-us')}>About</a></li>
+              <li><a href={this.renderMarketingSiteUrl('/enterprise')}>{siteName} for Business</a></li>
+              <li><a href={this.renderMarketingSiteUrl('/affiliate-program')}>Affiliates</a></li>
+              <li><a href={openSourceUrl}>Open {siteName}</a></li>
+              <li><a href={this.renderMarketingSiteUrl('/careers')}>Careers</a></li>
+              <li><a href={this.renderMarketingSiteUrl('/news-announcements')}>News</a></li>
+            </ul>
```

Comment:
Similar to my comment above regarding social-links, these links will also eventually need to be configurable - as some sites may not have all these links - and/or may not be based on their `MarketingSiteUrl`.